server/server.c

```diff
@@ -701,7 +704,7 @@ jd_daemon (void)
 }
 
 int
-main (int argc, char** argv)
+main(int argc, char** argv)
 {
```

Comment:
This contains a lot(!) of formatting changes.

Reply:
I will remove/reduce those formatting changes